src/portable/portable.c

```diff
@@ -92,6 +93,13 @@ static PortableMetadata *portable_metadata_new(const char *name, const char *pat
                         return mfree(m);
         }
 
+        /* The metadata file might have SELinux labels, we need to carry them and reapply them */
+        if (!isempty(selinux_label)) {
+                m->selinux_label = strdup(selinux_label);
+                if (!m->selinux_label)
+                        return portable_metadata_unref(m);
```

Comment:
if this error path is actually hit you'll cose stdin here...

Reply:
whops, good catch